src/components/diagrams/diagram-pane.js

```diff
@@ -314,14 +315,29 @@ export function DiagramPane({
             } else {
                 let singleLineDiagramView = createView(diagramState);
                 // if current view cannot be found, it returns undefined
-                // in this case, we remove it from diagram states
+                // in this case, we keep it in the diagram states and show a warning message inside the SLD
                 if (singleLineDiagramView) {
                     diagramViews.push({
                         ...singleLineDiagramView,
                         align: 'left',
                     });
                 } else {
-                    closeDiagramView(diagramState.id, diagramState.svgType);
+                    // because current view cannot be found, the id will be assigned to the name.
+                    const emptyDiagramView = {
+                        ...diagramState,
+                        name: diagramState?.id,
+                        // this variable is used to show a warning message inside the SLD
+                        isSvgNotFound: true,
+                        messageNotFound:
```

Comment:
It would be better to give this a more generic name, like "warningMessage"

Reply:
done